addons/web/static/tests/views/fields/many2many_tags_field.test.js

```diff
@@ -1541,3 +1542,43 @@ test("Many2ManyTagsField selected records still pickable and not duplicable", as
     await contains("a.o_delete").click();
     expect(".o_tag").toHaveCount(0);
 });
+
+test("Many2ManyTagsField with edit_tags option", async () => {
+    PartnerType._views = {
```

Comment:
There's no guarantee that you'll execute the mocked rpcs, so:

expect.assertions(4);